library/CMakeLists.txt

```diff
@@ -38,6 +38,20 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/include/hipsparse-version.h.in"
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/include/hipsparse.h"
                "${PROJECT_BINARY_DIR}/include/hipsparse/hipsparse.h" COPYONLY)
 
+set(HEADER_FILES 
+  internal/level1/hipsparse_axpyi.h
+  internal/level1/hipsparse_dotci.h
+  internal/level1/hipsparse_doti.h
+  internal/level1/hipsparse_gthr.h
+  internal/level1/hipsparse_gthrz.h
+  internal/level1/hipsparse_roti.h
+  internal/level1/hipsparse_sctr.h)
```

Comment:
Wouldn't be cleaner to do:

```
file(GLOB HEADER_FILES 
     "internal/level1/*.h"
)
```
?

If we merge all the PRs, we will have a significant number of lines in this file.

Reply:
I am not particularly worried about this but I can add a for loop once I start merging and resolving the merge conflicts.